port-forward-fun.conf

```diff
@@ -1,19 +0,0 @@
-##################### Notes ########################
-#                                                  #
-#  So, you wanna run a single Node server, huh?    #
-#  Here's what the Apache server might look like   #
-#                                                  #
-#  This will assume an Express server is already   # 
-#                running on Port 3000              #
-#                                                  #
-####################################################
-
-<VirtualHost *:80>
-    ServerAdmin          admin@mycoolsite.com
-    ServerName           mycoolsite.com
-    ProxyPreserveHost    On
```

Comment:
It would seem this line is what preserves the 'host' or URL. If it were turned off, our server wouldn't keep track of what site the request has come from.

Without that, we wouldn't know when to respond to mycoolsite.com with our XXX.XXX.XXX.XXX:3000 server!

http://httpd.apache.org/docs/current/mod/mod_proxy.html#proxypreservehost